docs/user/tutorials/01-50-expose-and-secure-a-workload/v2alpha1/01-52-expose-and-secure-workload-jwt.md

```diff
@@ -67,15 +67,15 @@ To access your HTTPBin Service, use [Postman](https://www.postman.com) or [curl]
 
 1. Try to access the secured workload without credentials.
```

Comment:
Sidenote, but why do we even provide a tutorial for Postman?

Reply:
@nataliasitko any comment/plans for this? this is out of scope for this task, but we may plan something.

Reply:
Yes, I'll take care of this in a follow-up task